README.md

```diff
@@ -8,8 +8,9 @@
 
 
 <p align="center">
-<img src="https://github.com/crowdsecurity/crowdsec/workflows/Go/badge.svg">
-<img src="https://github.com/crowdsecurity/crowdsec/workflows/build-binary-package/badge.svg">
+<img src="https://github.com/crowdsecurity/crowdsec/workflows/tests/badge.svg">
+<img src="https://github.com/crowdsecurity/crowdsec/workflows/build/badge.svg">
+<a href='https://coveralls.io/github/crowdsecurity/crowdsec?branch=unitest'><img src='https://coveralls.io/repos/github/crowdsecurity/crowdsec/badge.svg?branch=unitest' alt='Coverage Status' /></a>
```

Comment:
I think we need to change branch here before merge

Reply:
Fixed